index.html

```diff
@@ -13,7 +13,26 @@
 
 <body>
     <div class="container">
-   
+        <header>
+            
+            <nav>
+               <a href="index.html"><img class="logo" src="C:\Users\rache\Documents\Git\Sprint-Challenge--User-Interface\img\lambda-black.png" alt="black Lambda Logo">
```

Comment:
```suggestion
               <a href="index.html"><img class="logo" src="img/lambda-black.png" alt="black Lambda Logo">
```
This will be how you want to reference files in your project such as images and other pages. Remember: if you see "C:/Anything_at_all" listed as an image source like this, it should be changed. Let me know if you are having trouble with this, it can be tricky but it will be important going forward.